src/clean_air/visualise/generate_map_based_visualisations.py

```diff
@@ -98,3 +96,23 @@ def get_aircraft_track_map(aircraft_track_coords, output_path) -> map:
     m5.save(output_path)
 
     return m5
+
+def get_boundaries(boundary_data, output_path) -> map:
```

Comment:
This return type doesn't look valid to me because I can't find what `map` references. It looks like it maybe should be `folium.Map`
```suggestion
def get_boundaries(boundary_data, output_path) -> folium.Map:
```

Reply:
Done for all 3 functions